kimchi/src/circuits/polynomials/chacha.rs

```diff
@@ -549,6 +549,7 @@ mod tests {
             w: array::from_fn(|_| F::rand(rng)),
             z: F::rand(rng),
             s: array::from_fn(|_| F::rand(rng)),
+            coefficients: array::from_fn(|_| F::rand(rng)),
```

Comment:
Just a stupid question, why do we fill them up with random values when they are not used instead of zeros (or ones, or other value)?

Reply:
Oh, I am looking at the definition of coefficients inside ProofEvaluations now. Perhaps this is for zero knowledge, so that the proof is simulateable?

Reply:
my guess: because we do this for the other fields. We don't have a great strategy for testing anyway, we should implement a trait like this https://docs.rs/proptest/latest/proptest/arbitrary/trait.Arbitrary.html when we want random values